layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+if [ "$(id -u)" -ne 0 ]; then
+    echo "You may need to run this script as sudo if you see permission errors"
+    user=`whoami`
+fi
+if [ ! -z "${SUDO_USER}" ]; then
```

Comment:
I'm not sure about this variable ? Is it defined in env ?

Reply:
Yes, I believe so, when running as sudo, see https://linux.die.net/man/8/sudo.  It might not always work but it's not critical if we are just using it for messaging.  It worked in my case on Ubuntu 18.04

Reply:
Yes, this should be OK.  But if root executes this script, the `user` variable never gets set.

Reply:
good point, how about this logic update?